swagger-templates/src/main/java/com/okta/swagger/codegen/AbstractOktaJavaClientCodegen.java

```diff
@@ -155,8 +155,7 @@ protected void buildTopLevelResourceList(OpenAPI openAPI) {
                 String parent = (String) model.getExtensions().get("x-okta-parent");
                 if (parent != null) {
                     parent = parent.replaceAll(".*/", "");
-
-                    if (resources.contains(parent)) {
```

Comment:
tiny bug was causing a lot of the inheritance issues

Reply:
hm actually, looking through this again, my fix is also wrong. It should be adding the child if the parent already exists, not re-adding the parent